### DISCUSSION TITLE
Anyway to add legacy guess, guess scale or guess off?

### DISCUSSION THREAD

**Discussion by creeduk** [Mikubill/sd-webui-controlnet#1231] [Ideas]
Trying to reproduce and old result I ran into issues. 

First the new canny seems to be stricter, if that is the right way to describe it. It seems to make the generated image stick closer which can be good but can also stop some realism in result. To deal with that I got  closer with old model but still the image is a long way off. 
It feels like Balance is not enough, "ControlNet is more important" is too much I played with weight and this but then it leaves the control. 

**Sliding scale:**
This made me wonder if there was some way to get legacy mode or to have a sliding scale of Prompt to Controlnet rather than just the 3 modes? Maybe this would help get closer to old results but benefit from better control.

For me I am using --lowvram and I upgraded Automatic1111 which had remained paused on my machine for a couple of months. I know from other issues the vram setting can cause different results and maybe the auto1111 update added to this issue, still I felt maybe a sliding scale could help.

Not sure if it would benefit from that mode being off, I read it is basically guess mode which I never used on the old controlnet generations, no unsure if I can reproduce no guess mode with current settings.

**Comment by lllyasviel** [comment]
hello, we have tested that the balanced mode's outputs are exactly same with previous codes turning guess mode off if old 1.0 models are used. (All pixel RGB numbers are equal in extensive tests.)
If you can provide stronger evidences that some results mismatch, please open an issue and we will take a look. Sometimes people's feeling about past things are noy very accurate.
see also section "Archive of Deprecated Versions" in README to test old codes.
See also README to find that "balanced mode"=previous "Guess Mode" turned off.
